### PR TITLE
fix: Issue 76 update time format on start and end times

### PR DESCRIPTION
**Pull request by seanlawrenz**
fixes [issue 76](https://github.com/midohiofoodbank/freshtrak-public/issues/76)
Screen Shot 2020-04-23 at 12 06 41 PM

### CODE REVIEW

**Comment by seanlawrenz** [comment]
@nvengal I'm getting a `Permission to midohiofoodbank/freshtrak-pantry-finder-api.git denied to seanlawrenz.` when trying to push this, so I forked. Still can't seem to addd a reviewer so would you review this?

**Comment by nvengal** [comment]
> I'm getting a Permission to midohiofoodbank/freshtrak-pantry-finder-api.git denied to seanlawrenz. when trying to push this, so I forked. Still can't seem to add a reviewer so would you review this?

My bad I'll fix the reviewer permission issue. Forking the repo is fine. That's what I've been doing.

File: app/serializers/event_date_serializer.rb [hunk 1]
```diff
@@ -11,13 +11,23 @@ class EventDateSerializer < ActiveModel::Serializer
   attribute :start_time do
     start_time = object.start_time_key.to_s
     start_time = "0#{start_time}" if start_time.length == 3
-    Time.strptime(start_time, '%H%M').strftime('%I:%M %p')
+    minutes = start_time.split(//).last(2).join
```
Comment (nvengal) [inline app/serializers/event_date_serializer.rb:14]:
```suggestion
    minutes = start_time.last(2)
```

File: app/serializers/event_date_serializer.rb [hunk 2]
```diff
@@ -11,13 +11,23 @@ class EventDateSerializer < ActiveModel::Serializer
...
+    else
+      Time.strptime(start_time, '%H%M').strftime('%l:%M %p').lstrip
+    end
   end
 
   attribute :end_time do
     end_time = object.end_time_key.to_s
     end_time = "0#{end_time}" if end_time.length == 3
-    Time.strptime(end_time, '%H%M').strftime('%I:%M %p')
+    minutes = end_time.split(//).last(2).join
```
Comment (nvengal) [inline app/serializers/event_date_serializer.rb:25]:
```suggestion
    minutes = end_time.last(2)
```

Comment (seanlawrenz) [reply app/serializers/event_date_serializer.rb:25]: Lol. I'm such a ruby newb.

**Comment by davidpickering** [comment]
> 
> 
> > I'm getting a Permission to midohiofoodbank/freshtrak-pantry-finder-api.git denied to seanlawrenz. when trying to push this, so I forked. Still can't seem to add a reviewer so would you review this?
> 
> My bad I'll fix the reviewer permission issue. Forking the repo is fine. That's what I've been doing.

Did this get resolved? Need me to open anything up or give more permissions?

File: app/serializers/event_date_serializer.rb (hunk 2 above)
Comment (davidpickering) [reply app/serializers/event_date_serializer.rb]: Haha, thanks for jumping in on it!

**Pull request merged**